packages/xfsprogs/xfsprogs.spec

```diff
@@ -0,0 +1,93 @@
+Name: %{_cross_os}xfsprogs
+Version: 6.3.0
+Release: 1%{?dist}
+Summary: Utilities for managing the XFS filesystem
+License: GPL+ and LGPLv2+
+URL: https://xfs.wiki.kernel.org
+Source0: http://kernel.org/pub/linux/utils/fs/xfs/xfsprogs/xfsprogs-%{version}.tar.xz
+Patch1: 0001-libxfs-do-not-try-to-run-the-crc32selftest.patch
+
+BuildRequires: %{_cross_os}glibc-devel
+BuildRequires: %{_cross_os}libuuid-devel
+BuildRequires: %{_cross_os}libinih-devel
+BuildRequires: %{_cross_os}liburcu-devel
+BuildRequires: %{_cross_os}libblkid-devel
+
+Requires: %{_cross_os}liburcu
+Requires: %{_cross_os}libinih
+
+%description
+%{summary}.
+
+%package devel
+Summary: XFS filesystem-specific headers
+Requires: %{name}
+
+%description devel
+%{summary}.
+
+%prep
+%autosetup -n xfsprogs-%{version} -p1
+
+%build
+%cross_configure \
+  --enable-blkid=yes \
+  --enable-lto=no \
+  --enable-editline=no \
+  --enable-scrub=no
+
+%make_build
+
+%install
+make DIST_ROOT=%{buildroot} install install-dev \
+  PKG_ROOT_SBIN_DIR=%{_cross_sbindir} PKG_ROOT_LIB_DIR=%{_cross_libdir}
+
+rm -f %{buildroot}/%{_cross_libdir}/*.{la,a}
```

Comment:
Same as elsewhere, is there a flag you can use to disable compiling static libraries?